[Issue] Compile errors

## Description
The current repo default branch _develop_ does not compile. This is probably related to merging the last pull request [#56](https://github.com/SuessLabs/Linux.Bluetooth/pull/56).

`Restore complete (3.6s)
  Linux.Bluetooth netstandard2.0 failed with 3 error(s) and 75 warning(s) (6.7s)
/home/xxx/Linux.Bluetooth/src/Linux.Bluetooth/Extensions/WatchableExtensions.cs(64,48): error CS0308: The non-generic method 'IGattService1.GetAsync(string)' cannot be used with type arguments
    /home/xxx/Linux.Bluetooth/src/Linux.Bluetooth/Extensions/WatchableExtensions.cs(75,48): error CS0308: The non-generic method 'IGattCharacteristic1.GetAsync(string)' cannot be used with type arguments
    /home/xxx/Linux.Bluetooth/src/Linux.Bluetooth/Extensions/WatchableExtensions.cs(86,48): error CS0308: The non-generic method 'IGattDescriptor1.GetAsync(string)' cannot be used with type arguments
`

## Environment
* **OS Distro:**  _Ubuntu v24.04, etc._

## Severity (1-5)
5=High (_compile error_)

## Steps To Reproduce
Steps to reproduce the behavior:
1. git clone https://github.com/SuessLabs/Linux.Bluetooth.git
2. Linux.Bluetooth
3. dotnet build

--- a/src/Linux.Bluetooth/Extensions/WatchableExtensions.cs
+++ b/src/Linux.Bluetooth/Extensions/WatchableExtensions.cs
@@ -52,6 +52,7 @@ namespace Linux.Bluetooth.Extensions
     public static Task WaitForPropertyValueAsync<T>(this IBattery1 obj, string propertyName, T value, TimeSpan timeout)
       => WaitForPropertyValueInternalAsync(obj.GetAsync<T>, obj.WatchPropertiesAsync, propertyName, value, timeout);
 
+    /*
     /// <summary>Wait for GattService's Property and specified value to resolve.</summary>
     /// <typeparam name="T">Type of value.</typeparam>
     /// <param name="obj">GattService.</param>
@@ -84,6 +85,7 @@ namespace Linux.Bluetooth.Extensions
     /// <exception cref="TimeoutException">On timeout a <seealso cref="TimeoutException"/> is thrown.</exception>
     public static Task WaitForPropertyValueAsync<T>(this IGattDescriptor1 obj, string propertyName, T value, TimeSpan timeout)
       => WaitForPropertyValueInternalAsync(obj.GetAsync<T>, obj.WatchPropertiesAsync, propertyName, value, timeout);
+    */
 
     /// <summary>Wait for MediaControl's Property and specified value to resolve.</summary>
     /// <typeparam name="T">Type of value.</typeparam>

